record-editor/src/app/shared/config/authors.config.ts

```diff
@@ -80,6 +80,9 @@ export const authors: JsonEditorConfig = {
                 anchorBuilder: anchorBuilder,
               },
             },
+            institution: {
+              autocompletionConfig: affiliationAutocompletionConfig,
```

Comment:
did you test this locally? is it working?

Reply:
Yeah, tested and it works

Reply:
for me it looks correctly, maybe @oguzdemirbasci can have a look too?

Reply:
LGTM 👍